### PR TITLE
WIP: Public Key Announcement

### PR DESCRIPTION
**Pull request by PhillyWebGuy**

### CODE REVIEW

File: pallets/keystore/src/lib.rs [hunk 1]
```diff
@@ -56,9 +71,23 @@ pub mod pallet {
...
+            <IssuedKeys<T>>::insert(&who, &fingerprint, &location);
+
+            Self::deposit_event(Event::KeyExists(fingerprint, location, who));
+            Ok(())
+        }
+
         /// If a key needs to be removed from circulation, this extrinsic will handle deleting it
         /// and informing the network.
-        #[pallet::weight(10_000 + T::DbWeight::get().reads_writes(1,1))]
+        #[pallet::weight(26_623_000 + T::DbWeight::get().reads_writes(1,1))]
```
Comment (Romulus10) [inline pallets/keystore/src/lib.rs:90]: You can call the weights.rs functions directly in these pallet::weight proc macros. That way if you update weights, you can just copy over to weights.rs and don't have to do any line-swaps.

Comment (PhillyWebGuy) [reply pallets/keystore/src/lib.rs:90]: @Romulus10 Should I be able to just copy the generated file to weights.rs? I see the output doesn't quite look like what's in identity and trust. I think I'll push up my latest generated files shortly.

Comment (Romulus10) [reply pallets/keystore/src/lib.rs:90]: You should, yeah. I'll take a look at it once you push.

Comment (PhillyWebGuy) [reply pallets/keystore/src/lib.rs:90]: @Romulus10 Latest commit is weights.rs exactly as generated.

**Pull request merged**